backends/cxxrtl/cxxrtl_backend.cc

```diff
@@ -3263,6 +3263,7 @@ struct CxxrtlWorker {
 						debug_wire_type = wire_type; // wire is a member
 
 					if (!debug_alias) continue;
+					if (wire->port_input || wire->port_output) continue; // no aliases for inputs and outputs
```

Comment:
I think we could go with a more descriptive comment here, something like this:

```suggestion
					if (wire->port_input || wire->port_output) continue; // preserve input/output metadata in flags
```